ui/src/components/PollBody/PollBody.jsx

```diff
@@ -0,0 +1,338 @@
+import React, { useState, useEffect } from 'react';
+import AutoSuggest from 'react-autosuggest';
+import Voter from '../Voter/Voter';
+import AnswerOption from '../AnswerOption/AnswerOption';
+import API from '../../API';
+import members from '../../data/voters.json';
+import './PollBody.css';
+
+const PollBody = () => {
```

Comment:
When it comes to relatively complex state such as what you've been using here,
you may find use in `useReducer` rather than `useState`. useReducer allows you to streamline
state changes in more complex states which require multiple variables like this.

Reply:
Looked up useReducer - didn't know that before and it looks like a nice way to implement this. Thanks for the info!